src/libraries/Microsoft.PowerFx.LanguageServerProtocol/Protocol/SemanticTokensEncoder.cs

```diff
@@ -160,7 +162,8 @@ public static IEnumerable<uint> EncodeTokens(IEnumerable<ITokenTextSpan> tokens,
         /// <param name="currentLineNumber">Curent Line Number in the expression.</param>
         /// <param name="previousTokenLineNumber">Line number of the previous encoded token.</param>
         /// <param name="previousTokenStartIdx">Starting position of the previous encoded token.</param>
-        private static void EncodeAndAddToken(ICollection<uint> encodedTokens, ITokenTextSpan currentToken, uint? currentTokenStartLine, uint? currentTokenStartPosition, uint? endPosition, uint currentLineNumber,  ref uint previousTokenLineNumber, ref uint previousTokenStartIdx)
+        /// <param name="controlTokens">Collection to add control token data.</param>
+        private static void EncodeAndAddToken(ICollection<uint> encodedTokens, ITokenTextSpan currentToken, uint? currentTokenStartLine, uint? currentTokenStartPosition, uint? endPosition, uint currentLineNumber, ref uint previousTokenLineNumber, ref uint previousTokenStartIdx, ICollection<ControlToken> controlTokens = null)
```

Comment:
Make sure you test this in studio that semantic tokens are not affected at all by this (one off errors). While harmless changes, you never know.